finite_difference/src/global_parameters_mod.f90

```diff
@@ -16,17 +16,17 @@ module global_parameters_mod
   integer, parameter, public :: ALIGNMENT = 4
```

Comment:
Shouldn't ALIGNMENT also have a GO_? Also, NAME_LEN appears to be unused, is this needed as part of this library?

Reply:
I think I probably could get rid of NAME_LEN. Although I see that I've declared ALIGNMENT as public, it really should be 'internal' to the library, i.e. application code shouldn't need it and therefore I don't think I need to re-name it.

Reply:
Agreed, and then make it private as well ;)

Reply:
Done - I've moved it into `grid_mod` as that's the only place that uses it.